tests/test_schema.py

```diff
@@ -372,6 +373,29 @@ class Meta(PostSchema.Meta):
                 assert "from_context" in included["attributes"]
                 assert included["attributes"]["from_context"] == "Hello World"
 
+    def test_load_n_dump_same_schema(self):
+        json_data = {
+            "data": {
+                "type": "comments",
+                "id": "1",
+                "attributes": {"body": fake.bs()},
+                "relationships": {"author": {"data": {"type": "people", "id": "1"}}},
+            },
+            "included": [
+                {
+                    "type": "people",
+                    "id": "1",
+                    "first_name": fake.first_name(),
```

Comment:
These attributes should be inside the `"attributes": {}` key. I'm a tad surprised that this works.

Reply:
Corrected in the last commit